repose-aggregator/components/filters/versioning/pom.xml

```diff
@@ -63,14 +63,20 @@
             <version>1.3</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+            <scope>test</scope>
+        </dependency>
         <dependency>
-            <groupId>org.codehaus.jackson</groupId>
-            <artifactId>jackson-core-lgpl</artifactId>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-core</artifactId>
             <scope>test</scope>
```

Comment:
The `dependencyManagement` definition of the **Jackson** artifacts is in the parental lineage module `profile-support` POM. Since it does not include a **scope** element, the default of `compile` is assumed. If a child module re-defines the scope, it will override the default. In this particular case, the scope of the **Jackson** dependencies will be `test`.

```
$ cd <REPOSE_DIR>/repose-aggregator/components/filters/versioning
$ mvn dependency:tree
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building Repose Components - Versioning 7.0.1.2-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- maven-dependency-plugin:2.1:tree (default-cli) @ versioning ---
[INFO] org.openrepose:versioning:jar:7.0.1.2-SNAPSHOT
...
[INFO] +- com.fasterxml.jackson.core:jackson-annotations:jar:2.4.0:test (scope not updated to compile)
[INFO] +- com.fasterxml.jackson.core:jackson-core:jar:2.4.0:test (scope not updated to compile)
[INFO] +- com.fasterxml.jackson.core:jackson-databind:jar:2.4.0:test (scope not updated to compile)
...
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
```